server.go

```diff
@@ -124,6 +128,8 @@ func (s *Server) HandlePush(ctx context.Context, e *PushEvent) error {
 		return err
 	}
 
+	s.poster.Status(ctx, e, PendingAnalysisStatus)
```

Comment:
I saw above you post status only for `lookout.ReviewEvent`. But this is push event. Do you think it makes sense to call this method if we know it will always fail?

Reply:
Actually I think it doesn't make sense, but that's a conversation I want to open later.

The `s.post` below will also fail always for a push, so I think we should leave the `status` call also, and handle both things better later.